### PR TITLE
Add minimal example with plausible result

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: r/README.qmd [hunk 1]
```diff
@@ -34,3 +36,16 @@ bind_rows(
   plot(lwd = 5)
 ```
 
+
+```{r}
+x_joined_df = rnet_match(x, y, dist_tolerance = 20, slope_tolerance = 0.1, trees = "xy")
+x_aggregated_x = x_joined_df |>
+  group_by(j) |>
+  mutate(ij = y$value * shared_len) |>
```
Comment (JosiahParry) [inline r/README.qmd:44]: I think we need to subset y based on the indices in j and then multiply

Comment (Robinlovelace) [reply r/README.qmd]: Really? Happy to see this.

Comment (JosiahParry) [reply r/README.qmd]: try: mutate(ij = y$value[j] * shared_len)

Because right now y$value will recycle and ignore the index of j in our result

**Review by JosiahParry — APPROVED** [review]
going to merge. can revise later if need be

**Pull request merged**